book/src/troubleshooting.md

```diff
@@ -67,6 +67,20 @@ related to the failing operation.
 
 Try running commands with `RUST_LOG=debug` to get more information:
 
-```
+```shell
 RUST_LOG=debug kanidm login --name anonymous
 ```
+
+## Reverse Proxies not sending HTTP/1.1 requests
+
+NGINX (and probably other proxies) send HTTP/1.0 requests by default. This'll lead to errors like this in your proxy logs:
```

Comment:
A good addition would be "[..] requests, to the upstream server, by default.
This will/might improve search result quality for keywords "nginx kanidm upstream".

Reply:
Good catch, have added the words in case search engines filter out the text below it for matching.